src/renderers/testing/ReactTestRenderer.js

```diff
@@ -83,8 +86,11 @@ class ReactTestComponent {
 
   getPublicInstance(transaction: ReactTestReconcileTransaction): Object {
     var element = this._currentElement;
-    var options = transaction.getTestOptions();
-    return options.createNodeMock(element);
+    var options = this._hostContainerInfo;
+    if (options && options.createNodeMock) {
+      return options.createNodeMock(element);
+    }
+    return {};
```

Comment:
Can you throw if it is `null`? I think this might make it clear to Flow that it's not null.